app/assets/javascripts/active_admin.js

```diff
@@ -0,0 +1 @@
+//= require active_admin/base
```

Comment:
What is this for ?  we are now using webpacker instead of sprockets so shouldn't be using this